module/VuFind/src/VuFind/Db/Service/CommentsService.php

```diff
@@ -43,9 +47,13 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  */
-class CommentsService extends AbstractDbService implements CommentsServiceInterface, DbServiceAwareInterface
+class CommentsService extends AbstractDbService implements
+    CommentsServiceInterface,
+    DbServiceAwareInterface,
+    DbTableAwareInterface
 {
     use DbServiceAwareTrait;
+    use DbTableAwareTrait;
```

Comment:
I chose to use DbTableAwareTrait here because the service in #2233 doesn't have a custom factory, and I wanted to avoid creating one only to delete it later.